README.rst

```diff
@@ -279,6 +279,16 @@ List of supported endpoints
     # AI Generated Photos
     amadeus.media.files.generated_photos.get(category='MOUNTAIN')
 
+    # Trip Parser
+    # Encode to Base64 your booking confirmation file
+    response = amadeus.travel.trip_parser_jobs.post(amadeus.travel.from_file(file))
```

Comment:
file is actually the path to the file no? Should we specify which formats we support?